pipeline_dp/aggregate_params.py

```diff
@@ -262,10 +262,9 @@ def __post_init__(self):
                     "both max_partitions_contributed and "
                     "max_contributions_per_partition must be set.")
             elif n_not_none == 1:
-                raise ValueError(
-                    "AggregateParams: either none or both from "
-                    "max_partitions_contributed and "
-                    " max_contributions_per_partition must be set.")
+                raise ValueError("AggregateParams: either none or both from "
```

Comment:
nit: I would delete "from" ("either none or both max_partitions_contributed and ...")